plugin/test/make-source-from-operation.spec.ts

```diff
@@ -23,6 +27,146 @@ afterAll(() => {
   server.close();
 });
 
+describe("A production build", () => {
+  const bulkResult = { id: "gid://shopify/Product/12345" };
+
+  beforeEach(() => {
+    server.use(
+      graphql.query<CurrentBulkOperationResponse>(
+        "OPERATION_STATUS",
+        (_req, res, ctx) => {
+          return res.once(
+            ctx.data({
+              currentBulkOperation: {
+                id: ``,
+                status: `RUNNING`,
+              },
+            })
```

Comment:
It would be nice if we could treat these functions like fixtures. The typings make this tricky.